scanners/android/icon.go

```diff
@@ -25,30 +26,23 @@ func lookupIconName(manifestPth string) ([]icon, error) {
 	}
 
 	log.Debugf("Looking for app icons. Manifest path: %s", manifestPth)
-	parsedIcons, err := parseIconName(doc)
-	if err != nil {
-		return nil, err
-	}
-
-	return parsedIcons, nil
+	return parseIconName(doc)
 }
 
 // parseIconName fetches icon name from AndroidManifest.xml.
 func parseIconName(doc *etree.Document) ([]icon, error) {
 	man := doc.SelectElement("manifest")
 	if man == nil {
-		log.Debugf("Key manifest not found in manifest file")
-		return nil, nil
+		return nil, fmt.Errorf("Key manifest not found in manifest file")
 	}
 	app := man.SelectElement("application")
 	if app == nil {
-		log.Debugf("Key application not found in manifest file")
-		return nil, nil
+		return nil, fmt.Errorf("Key application not found in manifest file")
 	}
 	ic := app.SelectAttr("android:icon")
 	if ic == nil {
-		log.Debugf("Attribute not found in manifest file")
-		return nil, nil
+		// Gradle varaibles like ${appIcon} are not supported
+		return nil, fmt.Errorf("Attribute not found in manifest file")
```

Comment:
error string should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
NOTE: I think mentioning which attribute (`android:icon`) is missing would make this error more informative